vendor.conf

```
@@ -16,7 +16,7 @@ github.com/tchap/go-patricia                        a7f0089c6f496e8e70402f617336
 golang.org/x/net                                    0de0cce0169b09b364e001f108dc0399ea8630b3
 golang.org/x/sys                                    85ca7c5b95cdf1e557abb38a283d1e61a5959c31
 github.com/docker/go-units                          519db1ee28dcc9fd2474ae59fca29a810482bfb1 # v0.4.0
-github.com/docker/go-connections                    7395e3f8aa162843a74ed6d48e79627d9792ac55 # v0.4.0
+github.com/docker/go-connections                    7dc0a2d6ddce55257ea8851e23b4fb9ef44fd4a0
```

Perhaps add the `git describe -tags` output as a comment while we wait for v0.5.0 to be done, to give a bit of a clue what version we're vendoring.

```suggestion
github.com/docker/go-connections                    7dc0a2d6ddce55257ea8851e23b4fb9ef44fd4a0 # v0.4.0-19-g7dc0a2d
```